qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1 Send CV.feature

```diff
@@ -0,0 +1,218 @@
+Feature: 3.1 Send CV.feature
+
+Background:
+Given user opened the page with the form to send CV
```

Comment:
You should mention where this form is located (link), so provide more specific data for a user what and where to see (as I can open lot of sites with CV form)